build.sbt

```diff
@@ -28,7 +28,7 @@ lazy val root = (project in file("."))
     sbtPlugin := true,
     addSbtPlugin("com.thesamet" % "sbt-protoc" % "1.0.6"),
     libraryDependencies ++= Seq(
-      "com.thesamet.scalapb.zio-grpc" %% "zio-grpc-codegen" % "0.5.3",
+      "com.thesamet.scalapb.zio-grpc" %% "zio-grpc-codegen" % "0.6.1",
```

Comment:
Isn't this gonna force ZIO 2?

Reply:
Good point! 

You're right. The generated code in `zio-grpc` master (version 0.6.x) generates code containing zio 2, while the branch `0.5.x` generates code with dependency on `zio 1`

Closing this until we start using zio 2.